src/include/firebird/impl/inf_pub.h

```diff
@@ -184,7 +184,7 @@ enum info_features // response to fb_info_features
 	fb_feature_statement_timeout	= 6,	// Statement timeout is supported
 	fb_feature_statement_long_life  = 7,    // Prepared statements are not dropped on transaction end
 
-	info_provider_features_max      // Not really a feature. Keep this last.
+	info_features_max      // Not really a feature. Keep this last.
```

Comment:
I think all items of enums should have the same prefix.

Reply:
Yes but this one is not quite item of enum and used to be used in code like "info_features array[info_features_max]".

Reply:
It is still an item of the enum.

Reply:
As a compromise suggest fb_features_max